src/components/eventsPageComponents/DateFilter.jsx

```diff
@@ -1,7 +1,96 @@
-import React from "react";
+import dayjs from "dayjs";
+import React, { useState } from "react";
+import { GrFormNext, GrFormPrevious } from "react-icons/gr";
 
-function DateFilter() {
-    return <div>filter events by date</div>;
-}
+import { generateDate, months } from "../util/calendar";
+import cn from "../util/cn";
+
+const DateFilter = ({ onClick }) => {
+    const days = ["S", "M", "T", "W", "T", "F", "S"];
+    const currentDate = dayjs();
+    const [today, setToday] = useState(currentDate);
+    const [selectDate, setSelectDate] = useState(currentDate);
+
+    return (
+        <div className='flex justify-center items-center my-4'>
+            <div className='w-[90%] h-auto border rounded-lg  shadow-md'>
+                <div className='flex py-3 md:justify-between justify-center items-center px-6 rounded-t-lg bg-amber-500'>
+                    <h1 className='hidden md:flex select-none font-semibold w-2/3'>
+                        {months[today.month()]} {today.year()}
+                    </h1>
```

Comment:
i'd suggest we change the location of months and today while keeping the prev and next arrow in other words
Today        <  mm/yy  >

![image](https://github.com/202306-NEA-DZ-FEW/team-nine-capstone/assets/137968168/8fe4cebd-75c9-4ab9-a721-8c06ec3bc6b8)

Reply:
okay i will work on that

Reply:
it has been taken care of !